Makefile

```diff
@@ -66,5 +66,7 @@ print_src:
 	
 install:
 	@cp libwayuu.so /usr/lib
+	
+	@mkdir -p /usr/include/wayuu
 
-
+	@cp conf_parser.h file_utils.h http_utils.h json_utils.h log.h router.h string_utils.h wayuu.h wayuu_log.h ws.h /usr/include/wayuu
```

Comment:
Could we use a wildcard instead of enumerating the files one by one? It would make it more maintainable